### DISCUSSION TITLE
自带界面报错

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#12170] [A - General / Q&A]
1、自带的客户端界面，点击表会报错。
<img width="456" alt="image" src="https://user-images.githubusercontent.com/32891043/187329266-e2961b43-5a07-4440-a5bb-746af1d612b9.png">
2、通过sqlyou客户端工具，也是点击表报错，无法使用。
<img width="442" alt="image" src="https://user-images.githubusercontent.com/32891043/187329525-164e3a3e-3d94-4f61-abb1-7b0cffade229.png">
3、可视化界面能否优化？
4、部署方便能否优化？提供一些可视化部署工具，或者快速安装部署的shell脚本。

**Comment by yuanyuan8983** [comment]
What is your Doris version? Is command line execution OK?

**Reply by a3225167** [reply]
doris 1.1.1

**Comment by hf200012** [comment]
desc tablename all
